packages/fxa-profile-server/test/api.js

```diff
@@ -765,6 +767,28 @@ describe('api', function() {
           assertSecurityHeaders(res);
         });
     });
+
+    it('should omit `ecosystem_anon_id` if not returned by auth-server', () => {
```

Comment:
In the auth-server I've been slowly updating the test code to use `async/await` since it makes things a little easier to read.

Reply:
oh, nice! yeah, I was just trying to follow the existing style in the files, but I'll start converting to async/await :+1: